core/Plugin/Manager.php

```diff
@@ -1191,6 +1191,8 @@ private function executePluginDeactivate($pluginName)
      */
     private function unloadPluginFromMemory($pluginName)
     {
+        $this->unloadPlugin($pluginName);
+
```

Comment:
I think I don't understand the question. UnloadPluginFromMemory did before only remove the activated plugin, now it does actually remove the loaded and the activated plugin.